src/Components/DecisionCanvas/DecisionCanvas.jsx

```diff
@@ -84,6 +84,8 @@ function DecisionCanvas({ update, review, categoryData }) {
     }, {});
   }, [review]);
 
+  let a = 0;
+  let b = 0;
```

Comment:
Are these variables needed?

Reply:
yeah i don't know what these are for lol. good catch